plugin-modernizer-core/src/test/java/io/jenkins/tools/pluginmodernizer/core/recipes/UpgradeJenkinsVersionTest.java

```diff
@@ -93,6 +95,59 @@ void testPerformUpgradeWithoutBom() {
                  """));
     }
 
+    @Test
+    void testNoUpgradeIfJenkinsFileDoesNotSupportJava17() {
```

Comment:
No changes is done there. Since JDK 8 is still referenced and not supported by 2.452.4